go.mod

```diff
@@ -1,14 +1,16 @@
 module github.com/icinga/icinga-notifications
 
-go 1.21
+go 1.22
+
+toolchain go1.22.3
```

Comment:
Seems like the `toolchain` directive was added because Go hat to increase the `go` directive for you:

> For reproducibility, the go command writes its own toolchain name in a toolchain line any time it is updating the go version in the go.mod file (usually during go get).

-- https://go.dev/ref/mod#go-mod-file-toolchain

But I don't see a particular reason why we would need or want it here.